package.json

```diff
@@ -30,16 +30,21 @@
     "ember-cli": "^2.4.2",
     "ember-cli-blueprint-test-helpers": "^0.12.0",
     "ember-cli-dependency-checker": "^1.2.0",
+    "ember-cli-internal-test-helpers": "^0.7.0",
+    "ember-cli-is-package-missing": "^1.0.0",
+    "ember-router-generator": "^1.2.1",
```

Comment:
Yes, ember-cli dependencies (not devDependencies) should be able to be looked up directly in node-modules. My install is borked.